lib/src/pages/credits.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/material.dart";
+
+import "package:ramaz/widgets.dart";
+import "package:ramaz/data.dart";
+
+import "drawer.dart";
+
+/// Creates the Credits Page by iterating through each ContributorCard.
```

Comment:
```suggestion
/// Creates the Credits Page by creating a [ContributorCard] for each contributor.
```

When you put a type in `[]`, then the docs will have a link to that class.